Identity.BusinessLogic/Models/Register.cs

```diff
@@ -1,12 +1,31 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
 
 namespace Identity.Core.Models
 {
-    internal class Register
+    public class Register
     {
+        [Required(ErrorMessage = "Username field should not be empty")]
+        [MaxLength(20, ErrorMessage = "Username cannot be longer than 20 characters")]
+        [DisplayName("Username")]
+        public required string Username { get; init; }
```

Comment:
It is better to use FluentValidation instead of attributes to validate properties